docs/src/development/transfer-dedicated.md

```diff
@@ -1,25 +1,33 @@
 ---
-title: "Transferring data to and from a Dedicated cluster"
+title: "Transfer data to and from a Dedicated cluster"
 weight: 13
-sidebarTitle: "Syncing to Dedicated"
+sidebarTitle: "Sync to Dedicated"
 ---
 
-## Backing up staging and production files
+## Get the SSH username and host
```

Comment:
That is correct, an introduction seems like a good idea. I amended that in the latest commit.
I don't think that a how-to-guide would be the proper approach here since not all steps are required in each use case 🤔